Mathlib/Probability/Independence/Kernel.lean

```diff
@@ -788,8 +788,8 @@ theorem indepFun_iff_indepSet_preimage {mβ : MeasurableSpace β} {mβ' : Measur
   · rwa [← indepSet_iff_measure_inter_eq_mul (hf hs) (hg ht) κ μ]
 
 @[symm]
-nonrec theorem IndepFun.symm {_ : MeasurableSpace β} {f g : Ω → β} (hfg : IndepFun f g κ μ) :
-    IndepFun g f κ μ := hfg.symm
+nonrec theorem IndepFun.symm {γ : Type*} [MeasurableSpace β] [MeasurableSpace γ]
+    {f : Ω → β} {g : Ω → γ} (hfg : IndepFun f g κ μ) : IndepFun g f κ μ := hfg.symm
```

Comment:
`f` and `g` are already defined on different types in a `variable` line somewhere, so no need for `γ`:
```suggestion
nonrec theorem IndepFun.symm {_ : MeasurableSpace β} {_ : MeasurableSpace β'}
    (hfg : IndepFun f g κ μ) :
    IndepFun g f κ μ := hfg.symm
```
Same remark for the IndepFun.symm in Basic.

Could you do the same generalization for the lemma below? `IndepFun.ae_eq`